src/js/Content/Features/Store/App/FSteamDeckCompatibility.js

```diff
@@ -0,0 +1,52 @@
+import {ExtensionResources, HTML, Localization, SyncedStorage} from "../../../../modulesCore";
+import {Feature, RequestData} from "../../../modulesContent";
+
+export default class FSteamDeckCompatibility extends Feature {
+
+    checkPrerequisites() {
+        return SyncedStorage.get("showdeckcompat");
+    }
+
+    async apply() {
+
+        const compatNode = document.querySelector("[data-featuretarget=deck-verified-results]");
+        if (compatNode) {
+            // Move the section to a more prominent position
+            document.querySelector("#category_block").insertAdjacentElement("afterbegin", compatNode);
+            return;
+        }
+
+        // Compat info is not shown in regions where Valve doesn't sell the Steam Deck, or if the game is untested (usually means the status is unknown)
+        const data = await RequestData.getJson(`https://store.steampowered.com/saleaction/ajaxgetdeckappcompatibilityreport?nAppID=${this.context.appid}`);
+        if (!data || !data.success) { return; }
+
+        let status;
+        const code = data.results.resolved_category;
+        switch (code) {
+            case 0: // unknown, treating as unsupported since icons are the same
```

Comment:
How do you know that the icon is the same?
Also, shouldn't we at least display "Unknown" instead of "Unsupported" then?